scripts/zones/Ranguemont_Pass/mobs/Taisai.lua

```diff
@@ -3,29 +3,34 @@
 -- MOB:  Taisai
 -----------------------------------
 
+require("scripts/globals/status");
+local MobIDs = require("scripts/zones/Ranguemont_Pass/MobIDs");
+
 function onMobRoam(mob)
 
-    local Taisaijin = 17457216;
-    local Taisaijin_PH = 0;
-    local Taisaijin_PH_Table =
-    {
-        17457213,
-        17457214,
-        17457215
-    };
-    local Taisaijin_ToD = GetMobByID(Taisaijin):getLocalVar("ToD");
-
-    if (Taisaijin_ToD <= os.time()) then
-        Taisaijin_PH = math.random((0), (#Taisaijin_PH_Table));
-        if (Taisaijin_PH_Table[Taisaijin_PH] ~= nil) then
-            if (GetMobAction(Taisaijin) == 0) then
-                SetServerVariable("Taisaijin_PH", Taisaijin_PH_Table[Taisaijin_PH]);
-                DeterMob(Taisaijin_PH_Table[Taisaijin_PH], true);
-                DeterMob(Taisaijin, false);
-                DespawnMob(Taisaijin_PH_Table[Taisaijin_PH]);
-                SpawnMob(Taisaijin, "", 0);
-            end
+    local tts = GetServerVariable("Taisaijin_TTS");
+    if (tts > 0 and tts <= os.time() and GetMobAction(MobIDs.Taisaijin) == ACTION_NONE) then
```

Comment:
I'm confused, there is no table named MobIDs, there is a table named "tbl" inside the `MobIDs.lua` global for this zone. I see you local'd the require but the file itself isn't a table. Even if I was simply unawar that lua lets you treat files as table sin this manner, this is completely inconsistent with how every other MobIDs.lua has been used project wide. Are we going to update the other hundred+ entries? Standards **{**can I have it?**}**